apps/web/src/server/lib/redis.js

```diff
@@ -0,0 +1,21 @@
+import { init, get, set } from 'node-cache-redis';
```

Comment:
I think we should use the existing redis connection for session storage, and not create two connections to redis. We don't need this extra library, we've already got `redis` lib which gives us get and set functions. 

We should refactor so that we:
* parse conn string and create redis client (using existing lib)
* pass that client to session
* pass that client to msal plugin